### ISSUE TITLE
Confused - slowing down home bridge and 503 error

### ISSUE THREAD

**Issue by hajfly** [benzman81/homebridge-nukiio#100]
Installed the plugin successfully and it seemed to work, changed nothing but now only get following errors:

[18/01/2024, 17:18:38] [NukiBridge] Request to Nuki bridge 'http://XXX' finished with status code '503' and body 'HTTP 503 Unavailable'. null
[18/01/2024, 17:18:38] [NukiBridge] Request for lock state aborted. This is no problem and might happen due to canceled request or due to long response time of the Nuki bridge. Using cached value isLocked = 'true'.
[18/01/2024, 17:18:38] [NukiBridge] Request for lock state aborted. This is no problem and might happen due to canceled request or due to long response time of the Nuki bridge. Using cached value isLocked = 'true'.

Anything I can do here from my side?

**Comment by benzman81** [comment]
How many locks are connected to the bridge and what lock_state_mode are you using?

**Comment by hajfly** [comment]
one lock and one opener - always request lock states

**Comment by benzman81** [comment]
Try lock_state_mode = 1. Multiple device might cause this as the bridge can only handle one request at a time.

**Comment by hajfly** [comment]
As I use the settings I guess 1 is plugin cache?

**Comment by benzman81** [comment]
Yea

**Comment by hajfly** [comment]
Thanks, that works so far, no further error messages, what still doesn't work is, that I don't get a message on my phone as soon as the door is opened or closed, had this with the plugin I used before. Any further adjustments to make to get that ?

**Comment by benzman81** [comment]
This has nothing to do with the plugin as Homekit is responsible for the notifications. Maxbe you turned them off in home app.

**Comment by hajfly** [comment]
Thanks for the quick reply - strange I didn't and they are turned on. Well - thanks so far

**Comment by benzman81** [comment]
If you lock nuki while having the home app open, does it change to locked in the app? Or the other was around?

**Comment by hajfly** [comment]
yes it does but with quite a delay. When using the switches in the home app it also posts messages, but it doesn't when the lock is e.g. closed manually. The update in the app if done so takes up to two minutes.
